squidb/src/com/yahoo/squidb/utility/SquidUtilities.java

```diff
@@ -211,7 +205,6 @@ private static void copyStream(InputStream source, OutputStream dest) throws IOE
             }
 
             dest.write(buffer, 0, bytes);
-            dest.flush();
```

Comment:
Yeah it looks like the loop could be simplified a bit. Android doesn't offer the `Files` class though, and the channel-based version seems to be quite a bit slower, so just cleaning up the loop looks like the best option.